presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/confluent/ConfluentSchemaRegistryTableDescriptionSupplier.java

```diff
@@ -187,7 +190,11 @@ private String resolveSubject(String candidate)
     private KafkaTopicFieldGroup getFieldGroup(AvroSchemaConverter avroSchemaConverter, String subject)
     {
         try {
-            Schema schema = new Schema.Parser().parse(schemaRegistryClient.getLatestSchemaMetadata(subject).getSchema());
+            SchemaMetadata schemaMetadata = schemaRegistryClient.getLatestSchemaMetadata(subject);
+            if (schemaMetadata.getSchemaType() != "AVRO") {
```

Comment:
Once we move to latest schema registry can we handle them based on `SchemaProvider#getSchemaType` ? I will raising a PR to bind `SchemaProvider` and I guess we can inject it here also to validate here